pricing_import.py

```diff
@@ -1,419 +1,443 @@
-#!/usr/bin/env python2
-
+#!/usr/local/bin/python
```

Comment:
Isn't using env more flexible than having a static path to python here?

Reply:
Probably...I had modified that with the intent of including a docker-compose.yml and a python3 container.  Reverted.